### ISSUE TITLE
Add Inventory Kamera to list of scanners

### ISSUE THREAD

**Issue by kotx** [frzyc/genshin-optimizer#91]
IMHO it's faster than all the others listed

**Comment by D1firehail** [comment]
Copied from the Genshin Optimizer discord:

**Removal of Inventory Kamera from the Scanners Page**
A recent patch of IK caused a lot of issues for users, who flooded this discord with questions and bugreports. With the creator's consent, we have removed IK from the GO's scanners page, to prevent new users from having problems with IK. Once a new version of IK is released that resolves most of the current issues, IK will be reinstated on the page. Existing user of IK who has had no problem using the tool can continue to use it, there are no security issues with the tool.

---
Once the issue(s) have been resolved I would assume Inventory Kamera will be re-added to the list with the next GO update. 
I don't know if it's been fixed yet, but there hasn't been an update to GO. 
It may take a while since they're working on major changes to be able to properly deal with newer mechanics like Cinnabar Spindle weapon passive, Yun Jin etc.
